common/matchmaking/matchmaking-map-pools.ts

```diff
@@ -0,0 +1,57 @@
+import { Jsonify } from '../json'
+import { MapInfoJson, SbMapId } from '../maps'
+import { MatchmakingType } from '../matchmaking'
+
+export const MATCHMAKING_MAP_POOLS_LIMIT = 5
+
+export interface MatchmakingMapPool {
+  id: number
+  matchmakingType: MatchmakingType
+  maps: SbMapId[]
+  maxVetoCount: number
+  startDate: Date
+}
+
+export type MatchmakingMapPoolJson = Jsonify<MatchmakingMapPool>
+
+export function toMatchmakingMapPoolJson(pool: MatchmakingMapPool): MatchmakingMapPoolJson {
+  return {
+    id: pool.id,
+    matchmakingType: pool.matchmakingType,
+    maps: pool.maps,
+    maxVetoCount: pool.maxVetoCount,
+    startDate: Number(pool.startDate),
+  }
+}
+
+export function fromMatchmakingMapPoolJson(pool: MatchmakingMapPoolJson): MatchmakingMapPool {
```

Comment:
Do we need to do this? It's extremely uncommon that we have functions for this, we typically just leave them in JSON form